### ISSUE TITLE
IBBS Issues

### ISSUE THREAD

**Issue by GamerSymptoms** [NSSpiral/Blackb0x#56]
I have an Apple TV 2 that I'm trying to jailbreak using mac os 10.11. It gets through until it says sending IBSS then it prompts me to re-enter DFU. So i re-enter DFU then it again gets through until it says sending IBSS then Blackb0x crashes.

**Comment by NSSpiral** [comment]
Restart both the Apple TV and blackb0x and try again.

**Comment by GamerSymptoms** [comment]
Thank you for the quick response. Unfortunately I'm still having the same issue. I've tried it with and without the power cord connected. I restored the Apple TV a few times just to see if that would do it. Also i have deleted and redownloaded Blackb0x and it still crashes.

**Comment by NSSpiral** [comment]
It is an issue with the ibss loader for A5, if you restart both the ATV and
the blackb0x app it should eventually work. Or you could run the app in
terminal and paste me the output so I can see what exactly is wrong.

**Comment by GamerSymptoms** [comment]
I've made it past everything. Except now it stays stuck on waiting for ATV to boot. So i waited on that screen for about half an hour and decided to unplug the usb and connect my ATV to my TV. Waited another half hour and kodi never popped up. I then reconnected the usb opened blackb0x and it said jailbreak is already active. I then restarted the ATV and blackb0x and it prompted me to install jailbreak. I repeated these steps several times.
